.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: go
 
 go:
-  - 1.5
```

Comment:
I'm concerned about removing the 1.5 test and then never adding it back again. I think it's worth adding a conditional to https://github.com/Microsoft/vscode-go/blob/master/.travis.yml if that's all that needs to happen.